change_log/v9_0_0/FE-1587-pages-button.yaml

```diff
@@ -0,0 +1,3 @@
+Component Update: "Pages" component now uses new carousel component.
```

Comment:
Hey @damianOfKyan this is not correct `.yaml` formatting something like this might be better:
```
Component Update: "`Pages` component now uses new carousel component."
Bug Fixes: "Removed 'UNSAFE' legacy 'componentWillUpdate' from `Carousel` component. `Dialog Full Screen` component fixed UI bug with close icon not showing in various small screen sizes. Fixed `Heading` component back() function where now it will correctly accept onClick callback function."
Storybook Updates: "Added History feature to allow for steping back through the visited pages. Implemented Storybook knob for 'pageIndex' property. Back chevron button will step back in history of visited pages triggering sliding effect. Back chevron will now use/inherit themes and icon styling."
```

Reply:
thanks, updated.